src/uEMEP_auto_subgrid.f90

```diff
@@ -474,16 +475,16 @@ subroutine uEMEP_region_mask_new()
         integer region_mask_projection_type
         double precision region_mask_projection_attributes(10)
         ! the x and y coordinates
-        real, allocatable :: x_values_regionmask(:)
-        real, allocatable :: y_values_regionmask(:)
+        double precision, allocatable :: x_values_regionmask(:)
```

Comment:
Please change all occurrences of "double precision" to "real(dp)". "double precision" is legacy. Remember to add it to the use statement above.